cpp/src/io/avro/avro.hpp

```diff
@@ -62,13 +62,14 @@ struct column_desc {
  */
 struct file_metadata {
   std::map<std::string, std::string> user_data;
-  std::string codec       = "";
-  uint64_t sync_marker[2] = {0, 0};
-  size_t metadata_size    = 0;
-  size_t total_data_size  = 0;
-  size_t num_rows         = 0;
-  uint32_t skip_rows      = 0;
-  uint32_t max_block_size = 0;
+  std::string codec        = "";
+  uint64_t sync_marker[2]  = {0, 0};
+  size_t metadata_size     = 0;
+  size_t total_data_size   = 0;
+  size_type num_rows       = 0;
+  size_type skip_rows      = 0;
+  size_type total_num_rows = 0;
```

Comment:
Here we have `num_rows` and `total_num_rows` which are confusing. Can you add comments clarifying what is their difference please?

Reply:
Roger, will do.